DirectProgramming/C++SYCL_FPGA/Tutorials/Features/experimental/mmhost/src/mmhost.cpp

```diff
@@ -0,0 +1,149 @@
+#include <iomanip>
+#include <sycl/ext/intel/fpga_extensions.hpp>
+#include <sycl/ext/intel/prototype/interfaces.hpp>
+#include <sycl/ext/oneapi/annotated_arg/annotated_ptr.hpp>
+#include <sycl/sycl.hpp>
+#include "exception_handler.hpp"
+
+using namespace sycl;
+using namespace ext::oneapi::experimental;
+using usm_buffer_location =
+    ext::intel::experimental::property::usm::buffer_location;
+
+constexpr int kBL1 = 1;
+constexpr int kBL2 = 2;
+constexpr int kBL3 = 3;
+
+struct PointerIP {
```

Comment:
Something like this is better.
```suggestion
struct NonCustomizedArgumentInterfacesIP {
```